src/crimson/osd/main_config_bootstrap_helpers.h

```diff
@@ -21,14 +21,9 @@ void usage(const char* prog);
 
 inline uint64_t get_nonce()
 {
-  if (auto pid = getpid(); pid == 1 || std::getenv("CEPH_USE_RANDOM_NONCE")) {
-    // we're running in a container; use a random number instead!
-    std::random_device rd;
-    std::default_random_engine rng{rd()};
-    return std::uniform_int_distribution<uint64_t>{}(rng);
-  } else {
-    return pid;
-  }
+  std::random_device rd;
+  std::default_random_engine rng{rd()};
+  return std::uniform_int_distribution<uint64_t>{}(rng);
```

Comment:
Can this be `ceph::util::generate_random_number<uint64_t>()` for consistency?  Or even call `Messenger::get_random_nonce()` directly from crimson?